doc/manuals/admin/perf_tuning.md

```diff
@@ -112,6 +113,25 @@ on [the `notifQueue` block](statistics.md#notifqueue-block) may help you to tune
 
 [Top](#top)
 
+## Payload and message size and performance
+
+Orion Context Broker uses two defaults limits related with payloads and HTTP message size. In particular:
+
+* There is a default limit of 1MB in incoming HTTP request payload
+* There is a default limit of 8MB in outgoing HTTP request messages (including HTTP request line, headers and payload), which applies to notifications and forwarded requests
+
+This limit should suffice the most of the use cases and, at the same time, avoids denial of service due to
```

Comment:
should suffice the most => should suffice for most

Reply:
Fixed in e2d530b5c16a6a05f1f5c6d807f6cd33a3c76563